functions/date-past-now/1.0/index.js

```
@@ -0,0 +1,18 @@
+import moment from 'moment-timezone';
+
+const datePastNow = async ({ timeZone, dateTime }, steps) => {
+  const now = moment().tz(timeZone);
```

Maybe nice to validate if inputs are valid date, so we can throw a nice error